cmd/es-rollover/app/index_options.go

```diff
@@ -65,6 +65,12 @@ func RolloverIndices(archive bool, skipDependencies bool, prefix string) []Index
 		})
 	}
 
```

Comment:
I see that above is conditional on `!skipDependencies`
```
      --skip-dependencies           Disable rollover for dependencies index
```
I think we need a similar one for adaptive sampling, since not everyone will be using it. I would also make it skipped by default, otherwise the rollover might start failing for people after upgrade - they should explicitly opt-in for this. Suggestion:
```
      --adaptive-sampling           Enable rollover for adaptive sampling index
```

Reply:
and we should have unit test that checks the opt-in nature